gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/portal_page/exception/ParentNotFoundException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.apim.core.portal_page.exception;
+
+import io.gravitee.apim.core.exception.ValidationDomainException;
+
+public class ParentNotFoundException extends ValidationDomainException {
```

Comment:
I think we have a NotFound domain exception, to check...
```suggestion
public class ParentNotFoundException extends NotFoundDomainException {
```

Reply:
We do but since it does not inherit from ValidationDomainException, it results in a 500 :/

Reply:
not a 404 🤔 ?

Reply:
Done, fixed NotFoundDomainExceptionMapper as discussed